pkgs/by-name/wi/wivrn/package.nix

```diff
@@ -85,62 +117,121 @@ stdenv.mkDerivation (finalAttrs: {
     fi
   '';
 
-  nativeBuildInputs = [
-    cmake
-    git
-    glslang
-    pkg-config
-    python3
-  ] ++ lib.optionals cudaSupport [ autoAddDriverRunpath ];
+  nativeBuildInputs =
+    [
+      cmake
+      doxygen
+      git
+      glib
+      glslang
+      librsvg
+      pkg-config
+      python3
+      qt6.wrapQtAppsHook
+    ]
+    ++ lib.optionals cudaSupport [
+      autoAddDriverRunpath
+    ];
 
-  buildInputs = [
-    avahi
-    boost
-    cli11
-    eigen
-    ffmpeg
-    freetype
-    glm
-    harfbuzz
-    libdrm
-    libGL
-    libva
-    libX11
-    libXrandr
-    libpulseaudio
-    nlohmann_json
-    onnxruntime
-    openxr-loader
-    pipewire
-    shaderc
-    spdlog
-    systemd
-    udev
-    vulkan-headers
-    vulkan-loader
-    vulkan-tools
-    x264
-  ] ++ lib.optionals cudaSupport [ cudaPackages.cudatoolkit ];
+  buildInputs =
+    [
+      avahi
+      boost
+      bluez
+      cjson
+      cli11
+      dbus
+      eigen
+      elfutils
+      ffmpeg
+      freetype
+      glib
+      glm
+      gst_all_1.gst-plugins-base
+      gst_all_1.gstreamer
+      harfbuzz
+      hidapi
+      libbsd
+      libdrm
+      libdwg
+      libGL
+      libjpeg
+      libmd
+      libnotify
+      librealsense
+      libsurvive
+      libunwind
+      libusb1
+      libuvc
+      libva
+      libX11
+      libXrandr
+      libpulseaudio
+      nlohmann_json
+      opencv4
+      openhmd
+      openvr
+      openxr-loader
+      onnxruntime
+      orc
+      pipewire
+      qt6.qtbase
+      qt6.qttools
+      SDL2
+      shaderc
+      spdlog
+      systemd
+      udev
+      vulkan-headers
+      vulkan-loader
+      wayland
+      wayland-protocols
+      wayland-scanner
+      x264
+    ]
+    ++ lib.optionals cudaSupport [
+      cudaPackages.cudatoolkit
+    ];
 
   cmakeFlags = [
+    (lib.cmakeBool "WIVRN_USE_NVENC" cudaSupport)
     (lib.cmakeBool "WIVRN_USE_VAAPI" true)
+    (lib.cmakeBool "WIVRN_USE_VULKAN" true)
     (lib.cmakeBool "WIVRN_USE_X264" true)
-    (lib.cmakeBool "WIVRN_USE_NVENC" cudaSupport)
-    (lib.cmakeBool "WIVRN_USE_SYSTEMD" true)
     (lib.cmakeBool "WIVRN_USE_PIPEWIRE" true)
     (lib.cmakeBool "WIVRN_USE_PULSEAUDIO" true)
+    (lib.cmakeBool "WIVRN_FEATURE_STEAMVR_LIGHTHOUSE" true)
     (lib.cmakeBool "WIVRN_BUILD_CLIENT" false)
+    (lib.cmakeBool "WIVRN_BUILD_DASHBOARD" true)
+    (lib.cmakeBool "WIVRN_CHECK_CAPSYSNICE" false)
     (lib.cmakeBool "WIVRN_OPENXR_INSTALL_ABSOLUTE_RUNTIME_PATH" true)
```

Comment:
This is now `-DWIVRN_OPENXR_MANIFEST_TYPE=absolute` since https://github.com/WiVRn/WiVRn/commit/d7a1012e480e7f7e9852466263de83250f0d35bd